docs/configuring-datasets.md

```diff
@@ -22,9 +22,9 @@ as a function of time, stored in a [CSV file][csv].
 
 ```
 #time,value
-2018-01-01T00:00:00Z,0
-2018-01-01T00:00:01Z,1
-2018-01-01T00:00:02Z,2
+2018-01-01,0
```

Comment:
I simplified the time units since our temporal coverage will typically have day resolution (though most ISO formats should work).